tests/framework-tests/core/js/FluidViewTests.js

```diff
@@ -70,6 +70,18 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
             jqUnit.assertValue(message, node.jquery);
         };
 
+        fluid.tests.assertJQuery = function (message, node) {
+            jqUnit.assertValue(message, node.jquery);
+        };
+
+        fluid.tests.assertJQuery = function (message, node) {
+            jqUnit.assertValue(message, node.jquery);
+        };
+
+        fluid.tests.assertJQuery = function (message, node) {
+            jqUnit.assertValue(message, node.jquery);
+        };
+
```

Comment:
Unfortunately you'll need to back out the changes from this file before this pull request can be merged. There are some instructions in this tutorial about how to use `git checkout` to reset the changes in a single file. https://www.atlassian.com/git/tutorials/undoing-changes